resources/definitions/fdmprinter.def.json

```diff
@@ -1543,6 +1543,19 @@
                             "enabled": "roofing_pattern != 'concentric' and roofing_layer_count > 0 and top_layers > 0",
                             "limit_to_extruder": "roofing_extruder_nr",
                             "settable_per_mesh": true
+                        },
+                        "roofing_extension":
+                        {
+                            "label": "Top Surface Extension",
+                            "description": "Determines how much the top surfaces are extended beneath overlapping surfaces. By adjusting this value, you can ensure that the outer edges of the top surfaces are concealed by the layers above, resulting in a better visual quality, particularly for models with curved surfaces.",
+                            "type": "float",
+                            "default_value": "0.4",
+                            "value": "roofing_line_width * (wall_line_count_roofing + 1)",
+                            "minimum_value": "0",
```

Comment:
(might want to make this a wall-line width or half of that -- I'm not sure if we actually want this to be zero -- or even as a warning?)